dashboard/static/styles/main.styl

```diff
@@ -26,6 +26,11 @@ section {
     background-position: top center;
 }
 
+.top-bar-title h1 {
+    font-size: 1rem;
+    font-weight: normal;
```

Comment:
It would be, except foundation sets it to bold, which seems harder to read. So I'm over-riding Foundation's default `h1` style of `font-weight: bold` here.